xml/ses_glossary.xml

```diff
@@ -52,20 +115,79 @@
     </para>
    </glossdef>
   </glossentry>
-  <glossentry><glossterm>Monitor node, MON</glossterm>
+  <glossentry><glossterm>CRUSH rule</glossterm>
+   <glossdef>
+    <para>
+     The CRUSH data placement rule that applies to a particular pool(s).
+    </para>
+   </glossdef>
+  </glossentry>
+  <glossentry><glossterm>&dashboard;</glossterm>
+   <glossdef>
+    <para>
+     A built-in web-based &ceph; management and monitoring application to
+     administer various aspects and objects of the cluster. The dashboard is
+     implemented as a &mgr; module.
+    </para>
+   </glossdef>
+  </glossentry>
+  <glossentry><glossterm>&drvgrps;</glossterm>
+   <glossdef>
+    <para>
+     &drvgrps; are a declaration of one or more OSD layouts that can be mapped
+     to physical drives. An OSD layout defines how &ceph; physically allocates
+     OSD storage on the media matching the specified criteria.
+    </para>
+   </glossdef>
+  </glossentry>
+  <glossentry><glossterm>&grafana;</glossterm>
+   <glossdef>
+    <para>
+     Database analytics and monitoring solution.
+    </para>
+   </glossdef>
+  </glossentry>
+  <glossentry><glossterm>&mds;</glossterm>
+   <glossdef>
+    <para>
+     &mds; or MDS is the &ceph; metadata software.
+    </para>
+   </glossdef>
+  </glossentry>
+  <glossentry><glossterm>&mon;</glossterm>
+   <glossdef>
+    <para>
+     &mon; or MON is the &ceph; monitor software.
+    </para>
+   </glossdef>
+  </glossentry>
+  <glossentry><glossterm>&mgr;</glossterm>
    <glossdef>
     <para>
-     A cluster node that maintains maps of cluster state, including the monitor
-     map, or the OSD map.
+     &mgr; or MGR is the &ceph; Manager software, which collects all the state
```

Comment:
```suggestion
     &mgr; or MGR is the &ceph; manager software, which collects all the state
```

Reply:
You and @lproven have changed this back and forth. In theory, it should really be lowercase (which is what I had) and I will go with this as this is consistent with upstream.